### PR TITLE
CUMULUS-2704:Add option to create ORCA Reconciliation Report to dashboard

### PR DESCRIPTION
**Pull request by dopeters**
**Summary:** Summary of changes

Addresses [CUMULUS-2704: Add option to create ORCA Reconciliation Report to dashboard](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2704)

## Changes

* Add option to create ORCA Reconciliation Report to dashboard
* Requires @cumulus/api@10.1.1-alpha.7 published from cumulus core feature/rds-phase-2
* Update granule and reconciliation report fixtures due to productVolume type change from number to string

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Adhoc testing
- [ ] Integration tests

### CODE REVIEW

File: app/src/js/utils/type.js [hunk 1]
```diff
@@ -14,6 +14,11 @@ export const reconciliationReportTypes = [
     label: 'Inventory',
     description: 'This report allows you to select what bucket (s3, Cumulus, CMR) you would like to generate validation for all collections, granules and files for a duration.',
   },
+  {
+    id: 'Backup and Recovery', // TODO: verify that this matches the "type" returned from the API
+    label: 'Backup & Recovery',
+    description: 'This report compares the files that are in storage. This includes the Database versus what is in S3 Glacier/ORCA backup and recovery.'
+  }
```
Comment (dopeters) [inline app/src/js/utils/type.js:21]: This is the important change. It adds the "backup and recovery" option to the dropdown in the create rec report form

File: app/src/js/components/ReconciliationReports/orca-backup-report.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/**
```
Comment (jennyhliu) [inline app/src/js/components/ReconciliationReports/orca-backup-report.js:1]: This is a place holder for CUMULUS-2748

File: app/src/js/utils/table-config/reconciliation-reports.js [hunk 3]
```diff
@@ -252,3 +252,38 @@ export const tableColumnsGnf = [
     width: 50,
   },
 ];
+
+export const tableColumnsBackupAndRecovery = [
```
Comment (jennyhliu) [inline app/src/js/utils/table-config/reconciliation-reports.js:256]: This is a place holder for CUMULUS-2748

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## feature/rds-phase-2
+
+## Breaking Changes
+
+This version of the dashboard requires Cumulus API release from feature/rds-phase-2
```
Comment (kkelly51) [inline CHANGELOG.md:12]: The only thing is I'm wondering if we should wait until the branch is released then we'll have a specific version to mention

**Review by kkelly51 — APPROVED** [review]

**Pull request merged**